src/openrct2/network/NetworkBase.cpp

```diff
@@ -2567,7 +2567,7 @@ void NetworkBase::Server_Handle_AUTH(NetworkConnection& connection, NetworkPacke
         if (connection.AuthStatus == NetworkAuth::Verified)
         {
             const NetworkGroup* group = GetGroupByID(GetGroupIDByHash(connection.Key.PublicKeyHash()));
-            passwordless = group->CanPerformCommand(MISC_COMMAND_PASSWORDLESS_LOGIN);
+            passwordless = group->CanPerformCommand(static_cast<int32_t>(MiscCommand::PasswordlessLogin));
```

Comment:
function should take a `GameCommand`

Reply:
Those commands are unrelated to commands/game actions. I suppose the MISC_COMMANDS is a bit misleading as they are only used for networking the packets. There is one network packet that has this MISC_COMMANDS as some sort of sub packet type, its a bit silly.

Reply:
I agree the current setup isn't great but i don't see there being any issues doing it like this we can change it later